stories/notifications.stories.js

```diff
@@ -127,7 +127,7 @@ storiesOf( 'Global/Notifications', module )
 
 		return (
 			<WithTestRegistry callback={ setupRegistry }>
-				<UserInputSettings onCTAClick={ ( event ) => event.preventDefault() } />
+				<UserInputSettings isDismissable={ true } onCTAClick={ ( event ) => event.preventDefault() } />
```

Comment:
Related to my above comment, let's then use the new `UserInputPromptNotification` component here instead.

Reply:
👍